test/integration/BUILD

```diff
@@ -33,5 +33,21 @@ py_test(
         requirement("chardet"),
         requirement("certifi"),
         requirement("idna"),
+        requirement("pytest"),
```

Comment:
If more than one tests might use pytest, consider adding a macro to include all these dependencies.